src/activities/redraw/Redraw.qml

```diff
@@ -99,120 +100,127 @@ ActivityBase {
             spacing: 20
 
             // The color selector
-            Column {
-                id: colorSelector
-                Repeater {
-                    model: items.numberOfColor
-                    Item {
-                        width: 70 * ApplicationInfo.ratio
-                        height: width
-                        Image {
-                            id: img
-                            source: Activity.url + Activity.colorShortcut[modelData] + ".svg"
-                            sourceSize.width: parent.width
-                            z: iAmSelected ? 10 : 1
-
-                            property bool iAmSelected: modelData == items.colorSelector
-
-                            states: [
-                                State {
-                                    name: "notclicked"
-                                    when: !img.iAmSelected && !mouseArea.containsMouse
-                                    PropertyChanges {
-                                        target: img
-                                        scale: 0.8
+            Flickable{
+                interactive: true
+                width: 70 * ApplicationInfo.ratio
+                height: background.height
+                boundsBehavior: Flickable.StopAtBounds
+                contentHeight: items.numberOfColor * width
+                bottomMargin: bar.height
+                Column {
+                    id: colorSelector
+                    Repeater {
+                        model: items.numberOfColor
+                        Item {
+                            width: 70 * ApplicationInfo.ratio
```

Comment:
I would put an id to the Flickable and use flickable.width there. This way, if we change the flickable width for some reason, it will be the same here